checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
+
+class Vigenere(Test):
+
+    @check()
+    def exists(self):
+        """caesar.c exists."""
+        super().exists("caesar.c")
+
+    @check("exists")
+    def compiles(self):
+        """caesar.c compiles."""
+        self.spawn("clang -o caesar caesar.c -lcs50").exit(0)
+
+    @check("compiles")
+    def test_1a(self):
```

Comment:
No need for `test_` convention. Confer with @brianyu28 on how best to name methods moving forward.